### DISCUSSION TITLE
Anthropic RAG

### DISCUSSION THREAD

**Discussion by davide445** [FlowiseAI/Flowise#2062] [Show and tell]
I think it's nothing special, but I was searching on the marketplace such a solution due I find latest Claude 3 works well and fast, and wanted to have a RAG solution, so created this one

<img width="503" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/33984290/34fbc82d-a6e2-4233-bc05-f49ec4e0b0f0">

[Anthropic RAG Chatflow.json](https://github.com/FlowiseAI/Flowise/files/14803278/Anthropic.RAG.Chatflow.json)

In the end I have maintained the in-memory vector store due seems even with an 800k token document as input can work without problems on my 8GB RAM laptop.

Maybe worth adding it to the marketplace to just have any user a fast start on this

I will be happy finding a way to add a prompt on this

**Comment by HenryHengZJ** [comment]
thanks for contributing to the community!

**Comment by mcdonacp** [comment]
Did you have any luck working a prompt into the flow at all?
